recipes-core/underclock/underclock_1.0.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "When boot is done, underclock the CPU and GPU to save energy and tweak sysfs paths to allow for proper screen on/off transitions."
+PR = "r0"
+SRC_URI = "file://underclock.service \
+           file://underclock \
+           file://COPYING"
+LICENSE = "GPLv3"
+LIC_FILES_CHKSUM = "file://COPYING;md5=84dcc94da3adb52b53ae4fa38fe49e5d"
```

Comment:
If you use LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-3.0;md5=c79ff39f19dfec6d293b95dea7b07891" you don't have to ship your own GPLv3

Reply:
mhh I see that this is copy pasted from all the other layers :/ maybe we should ship one global recipe in meta-asteroid that has the systemd service and wiring for the script and let each watch provide the shell script only... but I won't block this PR on this because it's a pre-existing issue

Reply:
For reference sake, as discussed. We should probably replace the underclock script entirely in favor of a android-init based solution.
A systemd service can let android-init know that the system has booted using this script: https://github.com/mer-hybris/droid-hal-configs/blob/master/sparse/usr/lib/systemd/system/droid-late-start.service
And android-init should be able to handle this via: https://github.com/mer-hybris/android_system_core/commit/15344127b1f3f9712c1ef19e5fb0d0290b602f78